### PR TITLE
Fix security buffer overflows

### PR DESCRIPTION
**Pull request by LocutusOfBorg**
Hello, I fixed some possible security issues, and wrong returns to if statement.

Please review carefully, I didn't test them at all (and neither I do know if the printf parameters such as "name" still makes sense, or should just be dropped)

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -2,8 +2,8 @@ Source: anbox
 Section: misc
 Priority: optional
 Maintainer: Simon Fels <morphis@gravedo.de>
-Build-Depends: debhelper (>= 7), dkms, dh-systemd
-Standards-Version: 3.9.7
```
Comment (morphis) [inline debian/control]: Why is this bumped?  We're still supporting older platforms which may not have the latest version of the standard or debhelper

File: scripts/clean-build.sh [hunk 2]
```diff
@@ -55,9 +55,9 @@ cleanup
 
 mkdir build || rm -rf build/*
 cd build
-cmake ..
-make -j10
-make test
+cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
```
Comment (morphis) [inline scripts/clean-build.sh]: We want to use the default here and not RelWithDebInfo. If we want RelWithDebInfo as the default it has to go into CMakeLists.txt

File: debian/control (hunk 1 above)
Comment (morphis) [reply debian/control]: Hm, I was thinking debhelper was at that version for xenial. However thanks for pointing this out. Leaving it at "9" is right as we still want to support xenial.

File: scripts/clean-build.sh (hunk 2 above)
Comment (morphis) [reply scripts/clean-build.sh]: You're right that RelWithDebInfo helps with debugging etc. However as clean-build.sh is used as CI entry point we should build always with the defaults here. Lets set RelWithDebInfo as default in CMakeLists.txt then.

**Comment by LocutusOfBorg** [comment]
hello, I think I did them :)

**Pull request merged**